packages/deployment/upgrade-test/Dockerfile

```diff
@@ -88,10 +88,10 @@ SHELL ["/bin/bash", "-c"]
 RUN . ./upgrade-test-scripts/start_to_to.sh
 
 ARG DEST_IMAGE
-#this is agoric-upgrade-11 / vaults+1
-FROM ${DEST_IMAGE} as agoric-upgrade-11
+# this is agoric-upgrade-12 supporting wallets with non-vbank assets
+FROM ${DEST_IMAGE} as agoric-upgrade-12
 ARG BOOTSTRAP_MODE
-ENV THIS_NAME=agoric-upgrade-11 BOOTSTRAP_MODE=${BOOTSTRAP_MODE}
+ENV THIS_NAME=agoric-upgrade-12 BOOTSTRAP_MODE=${BOOTSTRAP_MODE}
```

Comment:
don't we need to add `FROM` for agoric-upgrade-11 ? I think `ghcr.io/agoric/agoric-sdk:35` is the upgrade-11 rc0 right now.